### PR TITLE
Add German language support to i18n.js

### PR DESCRIPTION
**Pull request by s-a**
Sorry for the commit noise. Autoformat removed some spaces. Hope you find this usefull.

### CODE REVIEW

**Comment by s-a** [comment]
yeay I passed the checks :)

File: src/js/i18n.js [hunk 1]
```diff
@@ -550,6 +550,97 @@ const _defs = {
...
+    appendTitleAuto: 'Ein neues Feld vom Typ \'auto\' hinzufügen (Strg+Umschalt+Ein)',
+    ascending: 'Aufsteigend',
+    ascendingTitle: 'Sortieren Sie die Elemente dieses ${Typs} in aufsteigender Reihenfolge',
+    actionsMenu: 'Klicken Sie zum Öffnen des Aktionsmenüs (Strg+M)',
+    cannotParseFieldError: 'Feld kann nicht in JSON geparst werden',
+    cannotParseValueError: 'Wert kann nicht in JSON geparst werden',
+    collapseAll: 'Alle Felder zuklappen',
+    compactTitle: 'JSON-Daten verdichten, alle Leerzeichen entfernen (Strg+Umschalt+\\)',
+    descending: 'Absteigend',
+    descendingTitle: 'Sortieren Sie die Elemente dieses ${Typs} in absteigender Reihenfolge',
```
Comment (josdejong) [inline src/js/i18n.js:569]: I think there is a typo and `${Typs}` should be `${type}`

File: src/js/i18n.js [hunk 2]
```diff
@@ -550,6 +550,97 @@ const _defs = {
...
+  },
+  de: {
+    array: 'Auflistung',
+    auto: 'Auto',
+    appendText: 'anhängen',
+    appendTitle: 'Fügen Sie nach diesem Feld ein neues Feld mit dem Typ \'auto\' ein (Strg+Umschalt+Ein)',
+    appendSubmenuTitle: 'Wählen Sie den Typ des neuen Feldes',
+    appendTitleAuto: 'Ein neues Feld vom Typ \'auto\' hinzufügen (Strg+Umschalt+Ein)',
+    ascending: 'Aufsteigend',
+    ascendingTitle: 'Sortieren Sie die Elemente dieses ${Typs} in aufsteigender Reihenfolge',
```
Comment (josdejong) [inline src/js/i18n.js:562]: I think there is a typo and `${Typs}` should be `${type}`

**Comment by josdejong** [comment]
Thanks a lot Stephan!

I see a handful occurrences of a typo `${Typs}` and `${Typ}`, that should be `${type}` if I'm not mistaken in order to get the actual type resolved. Can you look into that? Or is this on purpose?

**Comment by s-a** [comment]
I have 2 strings which do not translate in GUI 

`message: pair[0].type === 'object' ? 'Contains invalid properties' // object
        : 'Contains invalid items' // array` 

Did I miss these translation keys in dictionary or are they not translated in code?

**Comment by josdejong** [comment]
That's a good point, those messages are simply missing in the i18n.js and should be added there. I've added two new translation keys for those, see https://github.com/josdejong/jsoneditor/commit/2aee4a60081fe98e62787c30e1cbcc093b7016c8. 

Can you add the German translation for those two new keys?

**Comment by josdejong** [comment]
Thanks for the update Stephan, merging your PR now 👍

**Comment by s-a** [comment]
Thank you for your support. Do you have a approximately NPM release publish date in mind?

**Comment by josdejong** [comment]
I hope to do that tomorrow, maybe even tonight.

**Comment by josdejong** [comment]
@s-a your new German translation is now available in `v9.1.0`. Thanks again!

**Comment by s-a** [comment]
I have to thank you for your great support and this web component. The new release works perfect.

**Pull request merged**